CSharpRepl/CSharpRepl.csproj

```
@@ -27,7 +27,7 @@
   <ItemGroup>
     <PackageReference Include="PrettyPrompt" Version="4.1.1" />
     <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
```

Maybe could move to the stable version, but may need to handle some breaking changes

```suggestion
    <PackageReference Include="System.CommandLine" Version="2.0.0" />
```

---

Yes, I was looking into that... Although I managed to run it, many tests broke and I started to look into them to see exactly what needed to be updated... Might take some time